pkg/lib/page.css

```diff
@@ -1,5 +1,3 @@
-@import "../../node_modules/@patternfly/react-styles/css/patternfly-variables.css";
```

Comment:
Hah, fun, exactly the line I had to change this morning (so.. conflict?). How is it "globally accessible" now?

Reply:
See webpack config - sass-resources-loader.

Reply:
Basically I made sure that these three variables files I am defining there are accessible from anywhere. Imagine them as global variables.

Reply:
Thanks! I didn't know that mechanism.